src/sentry/static/sentry/app/utils/withPlugins.jsx

```diff
@@ -19,14 +19,35 @@ const withPlugins = WrappedComponent =>
       project: SentryTypes.Project,
     },
     mixins: [ProjectState, Reflux.connect(PluginsStore, 'store')],
+
     componentDidMount() {
+      this.fetchPlugins();
+    },
+
+    componentDidUpdate(prevProps) {
+      let organization = this.props.organization || this.getOrganization();
+      let project = this.props.project || this.getProject();
+
+      if (
+        (typeof prevProps.organization === 'undefined' ||
+          prevProps.organization.slug === organization.slug) &&
+        (typeof prevProps.project === 'undefined' ||
+          prevProps.project.slug === project.slug)
```

Comment:
What are we checking exactly? Why do we need to check for org as well - isn't the presence of a project enough? Setting variable/s might help readability of these 4 lines

Reply:
Updated with comments. We probably don't need to check for org, but doesn't hurt to be safe